cluster/cluster_test.go

```diff
@@ -44,7 +44,7 @@ func TestEncode(t *testing.T) {
 				3,
 				testutil.RandomETHAddress(),
 				testutil.RandomETHAddress(),
-				"0x00000002",
+				"0x90000069", // Sepolia testnet.
```

Comment:
use the constant

Reply:
will raise a PR to combine network chainID, name and forkVersion and replace relevant constants across the codebase